charts/fpl-case-service/values.yaml

```diff
@@ -1,6 +1,8 @@
 tags:
   fpl-ccd-stack: false
-  ccd-idam-pr: false
+
+idam-pr:
```

Comment:
i would remove this one as 
tags:
 fpl-ccd-stack: false
should disable entire ccd deployment